src/react-guides/Guides.tsx

```diff
@@ -523,4 +519,14 @@ export default class Guides extends React.PureComponent<GuidesProps, GuidesState
             selectedGuides:[],
         });
     }
+
+    private calcHorizontalTransform(offsetX: number, nextPos: number) {
+        const displayPos = [offsetX, nextPos];
```

Comment:
I also think instead of having two arguments and after combining them to the array, you can immediately pass them as an array
`this.calcHorizontalTransform([offsetX, nextPos]) `
...

`private calcHorizontalTransform(displayPos: number[]) {} `

Reply:
I also think name `positions` will sounds better than `displayPos`